automated install/basic-install.sh

```diff
@@ -1849,9 +1849,18 @@ get_binary_name() {
 
     # If the machine is aarch64 (armv8)
     if [[ "${machine}" == "aarch64" ]]; then
-        # If AArch64 is found (e.g., BCM2711 in Raspberry Pi 4)
-        printf "%b  %b Detected AArch64 (64 Bit ARM) architecture\\n" "${OVER}" "${TICK}"
-        l_binary="pihole-FTL-arm64"
+        if [[ "$(getconf LONG_BIT)" == "64" ]]; then
```

Comment:
Do we need to add `libc-bin` as new dependency? Must be `glibc-common` or so on Fedora.

Reply:
Hmm, yes, maybe. This is at the bare minimum requirements of the operating system, even any Alpine image has it by default. However, it also don't hurt adding the dependency explicitly.